packages/testing/src/api/index.ts

```diff
@@ -200,7 +200,13 @@ export function loadFirestoreRules(
 
   let client = new EMULATOR.FirestoreEmulator(
     FIRESTORE_ADDRESS,
-    grpc.credentials.createInsecure()
+    grpc.credentials.createInsecure(),
+    {
+      // Cap how much backoff gRPC will perform. This is testing code, so
+      // efficiency is less important than responsiveness.
+      'grpc.initial_reconnect_backoff_ms': 100,
+      'grpc.max_reconnect_backoff_ms': 100
+    }
```

Comment:
This was merged as part of https://github.com/firebase/firebase-js-sdk/pull/1403, not sure why it's showing up here...

Reply:
I think I know what's going on. I rebased the master yesterday after the release and the PRs are submitted before the rebase. Rebasing the PR should solve the problem.

Reply:
Hrm.  Rebasing master sounds problematic.  Can't we just do a normal merge of the release branch into master?

Reply:
Please **don't** rebase master. Everyone who has the repo checked out will have to perform a fix. For long-lived branches like master and our release branches, the only thing we can do is merge (and resolve conflicts).

Reply:
The problem with normal merge is that the version bump/release tag will appear at the HEAD, and it would be misleading to see commits that's not released appear before the release tag.

Reply:
Agreed it's not ideal and potentially dangerous. I'm open to change the process if we found better solution.

Reply:
I think just doing a normal (non-squash) merge would result in a sensible history.

Else, a squash merge with a clear description ("Merge release branch changes and version bump back into master.") would probably be sufficient.

Ultimately, we should be using the release branch and/or git tags to figure out what was actually in a given release.

Reply:
Right--the release branch should reflect the state of the release. The master branch reflects ongoing development that may have happened while that branch was coming about. If we merge the release branch back to master it will naturally show the history of when things happened.

Master is preparing for the next release.